wgsl/index.bs

```diff
@@ -4571,8 +4571,7 @@ textureSample(t : texture_depth_cube_array, s : sampler, coords : vec3<f32>, arr
   before sampling the texture. This offset is applied before applying any
   texture wrapping modes.<br>
   `offset` must be compile time constant, and may only be provided as a
-  [module constant](#module-constants), [literal](#literals) or
-  `const_expr` expression (e.g. `vec2<i32>(1, 2)`).<br>
+  [literal](#literals) or `const_expr` expression (e.g. `vec2<i32>(1, 2)`).<br>
```

Comment:
isn't literals a subset of `const_expr` expressions?

Reply:
Yes. I was trying to express this in a way that doesn't involve the reader having to understand the terminology of our grammar. I can remove it if it offends.

Reply:
ok, sounds like something the editors could resolve for us :)